src/plugins/dataflow/nodes/dataflow-node-plot.tsx

```diff
@@ -36,18 +36,9 @@ export const DataflowNodePlot: React.FC<INodePlotProps> = (props) => {
   const handleClickOffset = (zoomDir: Zoom) => {
     const max = maxY(props.data);
     const min = minY(props.data);
-    const difference = Math.abs(max - min);
-    const offset = 0.1 * difference;
-
-    if (zoomDir === Zoom.In ){
-      props.data.data.tickMax = max - offset;
-      props.data.data.tickMin = min + offset;
-    }
-
-    if (zoomDir === Zoom.Out){
-      props.data.data.tickMax = max + offset;
-      props.data.data.tickMin = min - offset;
-    }
+    const scalar = (zoomDir === Zoom.In) ? 1.25 : 0.8;
+    props.data.data.tickMax = scalar * max;
+    props.data.data.tickMin = scalar * min;
```

Comment:
This will not behave as expected for `tickMin > 0` (or if both values are negative, but that probably won't happen). Not sure what should happen in that case. Part of what @tealefristoe was recommending was an algorithm that worked independent of the values involved.